package.json

```
@@ -4,12 +4,9 @@
     "scripts": {
         "format": "prettier 'src/**/*.{js,html}' --write",
         "lint": "prettier --check 'src/**/*.{js,html}' && healthier 'src/**/*.{js,html}'",
-        "install": "grunt",
         "watch": "watch 'make assets' assets/",
         "pretest-legacy": "cd test_legacy/build && rollup -c",
-        "test-legacy": "./node_modules/mocha/bin/mocha --require reify",
-        "test": "ava && cd dw.js && npm install && npm test",
-        "test:watch": "ava --watch --verbose"
+        "test-legacy": "./node_modules/mocha/bin/mocha --require reify"
     },
```

```suggestion
        "test": "ava",
        "test:watch": "ava --watch --verbose"
    },
```
I think we want to keep the `ava` tests

---

Good catch. Didn't know that this repo had `ava` tests. Then we should restore the CI test job too.

---

They are back: aa004c1